deployment/ccip/changeset/test_environment.go

```diff
@@ -61,27 +62,28 @@ type TestConfigs struct {
 
 func (tc *TestConfigs) Validate() error {
 	if tc.Chains < 2 {
-		return fmt.Errorf("chains must be at least 2")
+		return errors.New("chains must be at least 2")
 	}
 	if tc.Nodes < 4 {
-		return fmt.Errorf("nodes must be at least 4")
+		return errors.New("nodes must be at least 4")
 	}
 	if tc.Bootstraps < 1 {
-		return fmt.Errorf("bootstraps must be at least 1")
+		return errors.New("bootstraps must be at least 1")
 	}
 	if tc.Type == Memory && tc.RMNEnabled {
-		return fmt.Errorf("cannot run RMN tests in memory mode")
+		return errors.New("cannot run RMN tests in memory mode")
 	}
 	return nil
 }
 
 func (tc *TestConfigs) MustSetEnvTypeOrDefault(t *testing.T) {
 	envType := os.Getenv(ENVTESTTYPE)
-	if envType == "" || envType == string(Memory) {
+	switch envType {
+	case string(Memory):
```

Comment:
You lost the default handling:
```suggestion
	case "", string(Memory):
```